ops/Dockerfile.local

```
@@ -0,0 +1,13 @@
+FROM golang:1.23.1-alpine3.20
+
+WORKDIR /app
+
+RUN go install github.com/air-verse/air@latest && \
+    go install github.com/a-h/templ/cmd/templ@latest
+
+COPY go.mod go.sum ./
+
+RUN go mod download
+
+CMD ["templ", "generate", "--watch", "--proxy=http://localhost:4000", \
+    "--open-browser=false", "-v", "--cmd=air -d -c .air-with-proxy.toml"]
```

add trailing newline
```suggestion
    "--open-browser=false", "-v", "--cmd=air -d -c .air-with-proxy.toml"]

```